system/velodyne_monitor/src/velodyne_monitor.cpp

```diff
@@ -32,14 +32,14 @@
 
 VelodyneMonitor::VelodyneMonitor() : Node("velodyne_monitor"), updater_(this)
 {
-  timeout_ = declare_parameter("timeout", 0.5);
-  ip_address_ = declare_parameter("ip_address", "192.168.1.201");
-  temp_cold_warn_ = declare_parameter("temp_cold_warn", -5.0);
-  temp_cold_error_ = declare_parameter("temp_cold_error", -10.0);
-  temp_hot_warn_ = declare_parameter("temp_hot_warn", 75.0);
-  temp_hot_error_ = declare_parameter("temp_hot_error", 80.0);
-  rpm_ratio_warn_ = declare_parameter("rpm_ratio_warn", 0.80);
-  rpm_ratio_error_ = declare_parameter("rpm_ratio_error", 0.70);
+  timeout_ = declare_parameter<double>("timeout");
+  ip_address_ = declare_parameter<string>("ip_address");
```

Comment:
```suggestion
  ip_address_ = declare_parameter<std::string>("ip_address");
```

Reply:
@karishma1911

Reply:
> @karishma1911

I fixed above issue, can you please go through this.
Thank you